services/src/layers/add_from_directory.rs

```diff
@@ -91,10 +83,12 @@ pub async fn add_layer_collections_from_directory<L: LayerDb>(db: &mut L, file_p
         }
         .validated()?;
 
-        db.add_collection_with_id(def.id, collection).await?;
+        // TODO: add only collections that aren't contained in any other collection to the root collection?
```

Comment:
could we add a "leftover" or "unsorted aka. your desktop"? collection for this?

Reply:
ddfafc8adfbc119ea5a6f0c02257dae460ce2468